idd/Energy+.idd.in

```diff
@@ -48456,6 +48456,7 @@ Fan:SystemModel,
        \type real
        \units W
        \autosizable
+       \default autosize
```

Comment:
Change Fan:SystemModel Design Electric Power to default to "autosize". Previously, if left blank, it fell through as zero which caused divide by zero when calculating the fan total efficiency.

Reply:
A very reasonable addition.